public/app/features/admin/AdminEditOrgPage.tsx

```diff
@@ -82,21 +88,18 @@ const AdminEditOrgPage = ({ match }: Props) => {
         <>
           <Legend>Edit organization</Legend>
           {orgState.value && (
-            <Form
-              defaultValues={{ orgName: orgState.value.name }}
-              onSubmit={(values: OrgNameDTO) => onUpdateOrgName(values.orgName)}
-            >
-              {({ register, errors }) => (
-                <>
-                  <Field label="Name" invalid={!!errors.orgName} error="Name is required" disabled={!canWriteOrg}>
-                    <Input {...register('orgName', { required: true })} id="org-name-input" />
-                  </Field>
-                  <Button type="submit" disabled={!canWriteOrg}>
-                    Update
-                  </Button>
-                </>
-              )}
-            </Form>
+            <form onSubmit={handleSubmit(onUpdateOrgName)} style={{ maxWidth: '600px' }}>
```

Comment:
Is this max-width something we can standardize? I believe we have multiple pages where some kind of width is being set to limit the input to being 100%. It is nothing blocking this PR, just something to think about :)

Reply:
Yeah, I've been wondering that myself, but not sure how to approach it. Maybe having a `Form` component in Grafana core could work 🤔